libs/blocks/gnav/gnav-search.js

```diff
@@ -18,10 +18,10 @@ const fetchResults = async (searchStr, locale = 'en_US') => {
   return null;
 };
 
-const getNoResultsEl = (value) => {
+const getNoResultsEl = (value, prefix, country = 'US') => {
```

Comment:
I'd suggest we either keep default values for both `prefix` and `country` or remove them for both, since the `getHelpxLink` method has defaults for both.